### DISCUSSION TITLE
cypress/grep issue using Typescript

### DISCUSSION THREAD

**Discussion by avi-ran** [cypress-io/cypress#29372] [Plugins]
Hi everyone :) 

I am currently trying to use cypress/grep plugin to efficiently filter my test cases.
Currently I am using a Typescript config on my cypress porject.

I tried to set up my project in order to use cypress/grep but with no success for instance...
I am following these instructions: https://github.com/cypress-io/cypress/tree/develop/npm/grep#install

I let you see my config using this minimal repro: https://github.com/avi-ran/cypress-grep-ts-issue/tree/main

I suspect this config to be wrong in `cypress/support/e2e.ts` (as I don't see any `registerCypressGrep` exported in the plugin code).

```
// "import" with `@ts-ignore`
// @see error 2306 https://github.com/microsoft/TypeScript/blob/3fcd1b51a1e6b16d007b368229af03455c7d5794/src/compiler/diagnosticMessages.json#L1635
// @ts-ignore
import registerCypressGrep from '@cypress/grep'
registerCypressGrep()
```

Let me know if there are any other mistake done on my config.

**Current behaviour on my minimal repro:**
- Two tests are run even if only one mentione 'config' tag

```bash
DEBUG=@cypress/grep npx cypress run --env grepTags=config

===============
  Running:  test_cases.cy.ts                                                               


  Dummy tests
    ✓ works as an array (20ms)
    ✓ works as a string (9ms)
```

**Expected behavior on my minimal repro:**
- Only the test with tag 'config' is run
```bash
DEBUG=@cypress/grep npx cypress run --env grepTags=config

===============
  Running:  test_cases.cy.ts                                                               


  Dummy tests
    ✓ works as an array (20ms)
```

Have a nice day!
Kind regards,
Aviran

**Comment by hatemeby** [comment]
Same issue here, anyone knows how to fix that?
